src/home/NotificationCenter.tsx

```diff
@@ -22,6 +27,73 @@ import { Spacing } from 'src/styles/styles'
 
 type NotificationsProps = NativeStackScreenProps<StackParamList, Screens.NotificationCenter>
 
+function useCleverTapNotifications() {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+
+  const messages = useSelector(cleverTapInboxMessagesSelector)
+
+  const notifications: Notification[] = []
+
+  for (const { messageId, header, text, icon, ctaText, ctaUrl, priority } of messages) {
+    const notificationId = `${NotificationType.clevertap_notification}/${messageId}`
+    const callToActions: CallToAction[] = [
+      {
+        text: ctaText,
+        onPress: (params) => {
+          ValoraAnalytics.track(HomeEvents.notification_select, {
+            notificationType: NotificationType.clevertap_notification,
+            selectedAction: NotificationBannerCTATypes.accept,
+            notificationId,
+            notificationPositionInList: params?.index,
+          })
+          CleverTap.pushInboxNotificationClickedEventForId(messageId)
+
+          const openInExternalBrowser = false
```

Comment:
sorry i didn't notice this before, i wonder if we should set this value from clevertap (and let it default to false if it's not there)? i'm wondering if there will be scenarios where it makes more sense to take the user out of the app, maybe for links related to NFTs it makes sense for iOS users to go out of the app to comply with app store guidelines?

Reply:
good catch, thanks! yes, we can add another optional tag for this case. i would appreciate naming advice for the tag. i'm thinking of simply `browser` (as opposed to "app") because tags should be typed manually and probably the simpler it is the less error-prone it is.

Reply:
ah, naming is always the hardest part 🙈 fwiw, i think we (maybe not us, maybe marketing team) will have some documentation on how to use homecards and in it we could define the tags available. so they can be copy + pasted, and we remove the need to have easily type-able names. `open:browser` seems fine, `openInExternalBrowser` was my first thought